src/components/Pages/PageGallery.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import MovieListConnect from '../Molecules/MovieList';
+import { setMovies } from '../../redux/actions';
+import './GalleryStyles.scss';
+import TemplateDefault from '../Templates/TemplateDefault';
+import LoadingAnim from '../Atoms/LoadingAnim';
+
+const Gallery = () => {
+  const dispatch = useDispatch();
+  const [isLoading, setIsLoading] = useState(true);
+  const [message, setMessage] = useState();
+
+  useEffect(() => {
+    const getMovies = async () => {
+      setIsLoading(true);
```

Comment:
I think this shouldn't be here. Does nothing in the logic flow as you call `setIsLoading` a few lines later.